### ISSUE TITLE
Tooltip error 

### ISSUE THREAD

**Issue by llcor3ll** [Gennoken/GennUI#63]
Latest version gives a tooltip error when hovering over any action bar, 

```
Message: ...nterface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua:525: attempt to call upvalue 'GetSpellTexture' (a nil value)
Time: Sun Aug 25 20:30:49 2024
Count: 2
Stack: ...nterface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua:525: attempt to call upvalue 'GetSpellTexture' (a nil value)
[string "=[tsm error check]"]:1: in main chunk
[string "=(tail call)"]: ?
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua"]:525: in function <...nterface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua:514>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `SetAction'
[string "@Interface/AddOns/ElvUI_Libraries/Core/LibActionButton-1.0/LibActionButton-1.0.lua"]:2674: in function `SetTooltip'
[string "@Interface/AddOns/ElvUI_Libraries/Core/LibActionButton-1.0/LibActionButton-1.0.lua"]:2378: in function <...ies/Core/LibActionButton-1.0/LibActionButton-1.0.lua:2370>
[string "@Interface/AddOns/ElvUI_Libraries/Core/LibActionButton-1.0/LibActionButton-1.0.lua"]:1254: in function <...ies/Core/LibActionButton-1.0/LibActionButton-1.0.lua:1252>
[string "=(tail call)"]: ?
[string "=[C]"]: ?

Locals: 
```

**Comment by llcor3ll** [comment]
![image](https://github.com/user-attachments/assets/d268171d-449b-46fb-bb07-7911ac48545b)
Cataclysm

**Comment by Aeon1985** [comment]
Still getting the error after updating:

Message: ...nterface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua:525: attempt to call upvalue 'GetSpellTexture' (a nil value)
Time: Wed Aug 28 08:03:50 2024
Count: 14
Stack: ...nterface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua:525: attempt to call upvalue 'GetSpellTexture' (a nil value)
[string "=[tsm error check]"]:1: in main chunk
[string "=(tail call)"]: ?
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua"]:525: in function <...nterface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua:514>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `SetAction'
[string "@Interface/AddOns/ElvUI_Libraries/Core/LibActionButton-1.0/LibActionButton-1.0.lua"]:2674: in function `SetTooltip'
[string "@Interface/AddOns/ElvUI_Libraries/Core/LibActionButton-1.0/LibActionButton-1.0.lua"]:2378: in function <...ies/Core/LibActionButton-1.0/LibActionButton-1.0.lua:2370>
[string "@Interface/AddOns/ElvUI_Libraries/Core/LibActionButton-1.0/LibActionButton-1.0.lua"]:1254: in function <...ies/Core/LibActionButton-1.0/LibActionButton-1.0.lua:1252>
[string "=(tail call)"]: ?
[string "=[C]"]: ?

Locals:

**Comment by Gennoken** [comment]
you can disable tooltip icons in GNUI option until i investigate and fix this

**Comment by Gennoken** [comment]
Fixed in 2.90, should be out soon
